<ui version="4.0" >
 <class>PopupOptionsClass</class>
 <widget class="OptionsPageI" name="PopupOptionsClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>333</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>PopupOptions</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QCheckBox" name="chkEnable" >
     <property name="text" >
      <string>Enable popups</string>
     </property>
     <property name="checked" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkRoundCorners" >
     <property name="text" >
      <string>Round corners</string>
     </property>
     <property name="checked" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>Classes</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" >
      <item>
       <layout class="QGridLayout" name="gridLayout_2" >
        <item row="0" column="2" >
         <layout class="QHBoxLayout" name="horizontalLayout_2" >
          <item>
           <widget class="QLabel" name="label_2" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Maximum" hsizetype="Preferred" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="whatsThis" >
             <string>:</string>
            </property>
            <property name="text" >
             <string>Timeout:</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spnTimeout" />
          </item>
         </layout>
        </item>
        <item row="1" column="2" >
         <layout class="QGridLayout" name="gridLayout" >
          <item row="0" column="0" >
           <widget class="QPushButton" name="btnTitleCol" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Maximum" hsizetype="Minimum" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string>Title color</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QPushButton" name="btnBgCol" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Maximum" hsizetype="Minimum" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string>Background color</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" >
           <widget class="QPushButton" name="btnTextCol" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Maximum" hsizetype="Minimum" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string>Text color</string>
            </property>
           </widget>
          </item>
          <item rowspan="2" row="2" column="1" >
           <widget class="QPushButton" name="btnIcon" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string>Icon</string>
            </property>
            <property name="iconSize" >
             <size>
              <width>64</width>
              <height>64</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QWidget" native="1" name="widTitleCol" >
            <property name="autoFillBackground" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QWidget" native="1" name="widBgCol" >
            <property name="autoFillBackground" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0" >
           <widget class="QWidget" native="1" name="widTextCol" >
            <property name="autoFillBackground" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="0" colspan="2" >
           <widget class="QPushButton" name="btnPreview" >
            <property name="text" >
             <string>Preview</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item rowspan="2" row="0" column="1" >
         <widget class="QListWidget" name="lstClasses" />
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>OptionsPageI</class>
   <extends>QWidget</extends>
   <header>options_i.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
